product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Security"
+---
+
+EDB's cloud offering runs in your own cloud account, isolates your data from other users, and gives you control over our access to it. The key security features are:
```

Comment:
This sentence feels like it could use some marketing spin - it reads like it's there because we had to put something at the top before the list. This page is essentially marketing; we should make it pop a bit.

Reply:
@ba-edb We could borrow this, slightly reworked, from the pitch deck: EDB Cloud is a fully managed cloud database from the Postgres experts. The key security features are: ...
Too much?